src/main/java/ladder/domain/Results.java

```diff
@@ -0,0 +1,38 @@
+package ladder.domain;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+
+public class Results {
+    private final Map<String, String> result;
+    private List<Result> results;
+
+    public Results(Map<String, String> result) {
+        this.result = new HashMap<>(result);
+    }
+
+    public Map<String, String> getResult() {
+        return result;
+    }
+
```

Comment:
키, 밸류 모두 String 타입이라

타입을 사용하는 측에서는 키가 무엇이고, 밸류가 무엇인지 바로 파악하기가 어려울 것 같아요

함수 네이밍을 `nameToPrizeResults` 같은 표현으로 나타내도 좋고
javadoc을 이용해서 어떤 값이 반환되는지 작성하면 좋을 것 같아요
```suggestion
    /**
     * 참가자와 경품의 매칭 결과를 맵으로 반환
     *
     * @return 참가자 이름을 키로, 상품 이름을 값으로 갖는 맵
     */
    public Map<String, String> getResult() {
        return result;
    }

```